includes/admin/class-wc-admin-notices.php

```diff
@@ -49,7 +49,9 @@ public static function init() {
 		add_action( 'switch_theme', array( __CLASS__, 'reset_admin_notices' ) );
 		add_action( 'woocommerce_installed', array( __CLASS__, 'reset_admin_notices' ) );
 		add_action( 'wp_loaded', array( __CLASS__, 'hide_notices' ) );
-		add_action( 'shutdown', array( __CLASS__, 'store_notices' ) );
+		if ( ! WC_Install::is_new_install() || ! wc_is_running_from_async_action_scheduler() ) {
```

Comment:
Probably add a comment here about why because it won't be super clear otherwise

Reply:
Good idea, added.